src/dataset/domain/models/Dataset.ts

```diff
@@ -16,10 +16,23 @@ export interface License {
   uri: string
   iconUrl?: string
 }
+export interface Citation {
+  authors: string[]
+  creationYear: number
+  title: string
+  persistentIdentifier: string
+  persistentIdentifierUrl: string
+  publisher: string
+  version: string
+  UNF?: string
+  isDeaccessioned?: boolean
+}
 export interface Dataset {
   id: string
   title: string
   labels: DatasetLabel[]
+  version: string
```

Comment:
Were are we using version: string? Can we remove it?

Reply:
Since the version refers to the Dataset and not specifically to the Citation, I think it makes sense to keep it, as you mentioned above.